LICENSE.MIT

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Pyvec <http://pyvec.org>
+Copyright (c) 2019 Pyvec, o.s. <info@pyvec.org>
```

Comment:
`z.s.`?

Reply:
Where is the facepalm emoji when I need it 😄

Reply:
Fixed directly in master using GitHub clicking ✏️